bindings/ios/MEGASdk.h

```diff
@@ -8695,6 +8695,32 @@ typedef NS_ENUM(NSInteger, AffiliateType) {
 */
 - (void)sendEvent:(NSInteger)eventType message:(NSString *)message;
 
+#pragma mark - Banner
+
+/**
+ * @brief Requests a list of all Smart Banners available for current user.
+ *
+ * The response value is stored as a MegaBannerList.
+ *
+ * The associated request type with this request is MegaRequest::TYPE_GET_BANNERS
```

Comment:
We need to convert the object types in MEGAAPI to our binding objects. Like MEGARequest, MEGABannerList, MEGAError, etc.